experiments/induction_results.py

```diff
@@ -47,10 +47,20 @@ def set_plotly_renderer(renderer="emacs"):
 all_runs = api.runs(path="remix_school-of-rock/acdc", filters={"group": ACDC_GROUP})
 
 df = pd.DataFrame()
+
```

Comment:
Thank you for updating this! I don't think we're using this notebook anymore though, it's all `roc_plots` or something, so I'm gonna delete this :(